app/assets/stylesheets/skills_page.scss

```diff
@@ -0,0 +1,32 @@
+$base        : #AAAAAA;
+$secondary-color: white;
+
+.side-bar {
+  background-color: $secondary-color;
+  border: 25px $secondary-color solid;
+  border-radius: 6px;
+  margin-top: 20px;
+  padding: 0;
+}
+
+#fl-img-sidebar {
+border-bottom-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 0 spaces